### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Aremusch**
### Created the Homepage on _/services_

Home Page preview (WIP) ⬇️
Capture d’écran 2024-04-30 à 16 09 59

### 9dc18ac
created following file to stylesheets/pages : 
_index.scss

created following file to stylesheets : 
application.scss

created following file to stylesheets/config :
_fonts.scss

created following files to stylesheets/pages : 
_index.scss

### 57f89e8
created following files to stylesheets/pages : 
_home.scss (_and linked dit to _index.scss_)

### 1d4f0a3
created following files to components.scss : 
_searchbar.scss
_services-grid.scss

### 054d0bc
created following files to components.scss : 
_banner.scss
_trendings-grid.scss
added following images to assets/images: 
avatar.webp
banner2.png
persona.png

### a6dfd1b
Added some hover effects on service-grid and treding-grid HTML elements.
changed the placeholder of the search bar from "`How can we help you?`" to "`What are you looking for today?`"

### e7db039
merged from master

### af730d5
Created homepage#index's view on `homepage_controller.rb` 
Switched the homepage#index's view to `services_controller.rb`
Deleted the following files : 
views/**homepage/index.html.erb**
app/controllers/**homepage_controller.rb**
Deleted the `root to: 'homepage#index'` in **routes.rb**

### CODE REVIEW

File: app/views/Homepage/index.html.erb [hunk 1]
```diff
@@ -1,10 +1,37 @@
-<div class="banner">
-  <h1>Rent a Peer</h1>
-  <h3>Catchy tagline</h3>
+<div class="navbar justify-content-center"><h1>THIS IS A NAVBAR</h1></div>
+<%# ABOVE NAVBAR SHOULD BE REPLACED BY RUCHA'S WORK %>
```
Comment (williamgrant04) [inline app/views/Homepage/index.html.erb]: This is rendered in `layout/application.rb` anyway so no need to specify the div

File: app/views/Homepage/index.html.erb [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+  <%# END OF CODE FOR THE SEARCH BAR %>
+  <%# HERE IS THE CODE FOR THE SERVICES GRID ⬇️ %>
+  <%# <% for each services do |service| %>
+  <div class="services">
+    <div class="grid">
+      <div class="card--services service">
+        <div class="bg--illustration">
+        </div>
+          <div class="display--indicator service-indicator"><i class="fa-solid fa-screwdriver-wrench"></i></div>
+          <%# WE SHOULD DEFINE ONE ICON PER DEFINED SERVICE %>
```
Comment (williamgrant04) [inline app/views/Homepage/index.html.erb]: Good idea, we should probably put that in the `services` table in the database to fetch them dynamically

File: app/views/Homepage/index.html.erb [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
+      </div>
+      <p>Mechanic</p>
+      </div>
+  </div>
+  <%# <% end %>
+  <%# END OF CODE FOR THE SERVICES GRID %>
 </div>
+<div class="justify-content-center"><h1>THIS IS A FOOTER</h1></div>
```
Comment (williamgrant04) [inline app/views/Homepage/index.html.erb]: I don't think we should have a footer

File: test/controllers/homepage_controller_test.rb
Comment (bdoughertys) [inline test/controllers/homepage_controller_test.rb:1]: We can probably delete this file.

File: app/helpers/homepage_helper.rb
Comment (bdoughertys) [inline app/helpers/homepage_helper.rb:1]: We can probably delete this file.

File: app/views/Homepage/index.html.erb [hunk 4]
```diff
@@ -20,18 +18,58 @@
...
   <%# HERE IS THE CODE FOR THE SERVICES GRID ⬇️ %>
   <%# <% for each services do |service| %>
-  <div class="services">
-    <div class="grid">
-      <div class="card--services service">
-        <div class="bg--illustration">
+  <div class="bottom-container d-flex">
+  <%# HERE IS THE CODE FOR THE TRENDINGS ⬇️ %>
+    <div class="trendings w-25">
+    <%# trending 1 %>
```
Comment (williamgrant04) [inline app/views/Homepage/index.html.erb]: These will probably be rendered dynamically too, but don't change this for now.

File: app/assets/stylesheets/components/_services-grid.scss [hunk 5]
```diff
@@ -1,20 +1,33 @@
...
   border-radius: 50%;
-  width: 80px;
-  height: 80px;
-  line-height: 80px;
+  width: 50px;
+  height: 50px;
+  line-height: 50px;
   i {
     margin: auto;
   }
```
Comment (williamgrant04) [inline app/assets/stylesheets/components/_services-grid.scss]: Fix indentation on lines 21-27

**Review by williamgrant04 — CHANGES_REQUESTED** [review]
Only need to change indentation error for now, otherwise looks good.

File: app/views/Homepage/index.html.erb (hunk 4 above)
Comment (Aremusch) [reply app/views/Homepage/index.html.erb]: yep, i just created those to check to display :) when we'll have the require controllers#actions, i'll make it dynamic

**Review by williamgrant04 — APPROVED** [review]
Looks good

**Review by williamgrant04 — CHANGES_REQUESTED** [review]
Should route homepage to root path so that when we push to heroku there's something other than the default rails thing

**Review by williamgrant04 — APPROVED** [review]
👍

**Pull request merged**